### ISSUE TITLE
ModuleNotFoundError: No module named 'exllamav2'

### ISSUE THREAD

**Issue by lambjohn36** [Zuellni/ComfyUI-ExLlama-Nodes#17]
Hi, I'm very new to all this. I've been messing around trying to install your custom node but I keep getting this:
  ![Screenshot 2024-02-21 155702](https://github.com/Zuellni/ComfyUI-ExLlama-Nodes/assets/160782942/a0c4d4f7-f83a-45af-ac7f-9305f57d423c)

Some more info that might help, I got : Python 3.11.8, Torch 2.2.0+cu118, CUDA 11.8
The last wheel I've tried: exllama-0.0.18+cu118-cp311-cp311-win_amd64.whl

**Comment by Zuellni** [comment]
Are you sure you're installing the correct wheel? These nodes require ExLlamaV2 as of a couple months now, ExLlama(V1) is deprecated.

This *should* work for you but I'm not sure about the compatibility of the wheels for Torch 2.2 and CUDA 11.8, there were some problems with them before. You might have to upgrade your Torch install to 2.2+cu121 if you still get errors.
```
.\python_embeded\python.exe -m pip install -U https://github.com/turboderp/exllamav2/releases/download/0.0.13.post2/exllamav2-0.0.13.post2+cu118-cp311-cp311-win_amd64.whl
```

**Comment by lambjohn36** [comment]
Okay I've finally managed to get my torch install to what you said:
![Screenshot 2024-02-26 001633](https://github.com/Zuellni/ComfyUI-ExLlama-Nodes/assets/160782942/9bf66086-5774-462d-b278-8fddd58e42bf)
But I still get the same error. I've tried a bunch of different versions. I now know why I ended up installing some exllamav1: on comfyui manager, in your Node's description there is a link that points to a list of exllamav1. Updating it might help people like me who don't pay attention ;). 

![Screenshot 2024-02-26 002035](https://github.com/Zuellni/ComfyUI-ExLlama-Nodes/assets/160782942/cd60d833-f905-46ed-971f-9f9df25ca8a1)

I do have exllamav2 now, hopefully the correct version: 

![Screenshot 2024-02-26 003251](https://github.com/Zuellni/ComfyUI-ExLlama-Nodes/assets/160782942/d9d5a437-397b-4881-86b3-2bcc42378e13)

This is what I get: 

![Screenshot 2024-02-26 0030551](https://github.com/Zuellni/ComfyUI-ExLlama-Nodes/assets/160782942/4f52dd23-15b3-4fd3-a052-4c0a6a625af2)

Thanks for helping me, I'm really interested  in trying your nodes.
